Multisig/UI/Onboarding/TermsView.swift

```diff
@@ -15,9 +15,9 @@ struct TermsView: View {
     @State private var showPrivacyPolicy = false
     @State private var showTerms = false
 
-    private let topPadding: CGFloat = 24
-    private let bottomPadding: CGFloat = 20
-    let interItemSpacing: CGFloat = 12
+    private let topPadding: CGFloat = Spacing.XL
```

Comment:
Can we please change the short names "XL" "S" "L"

Reply:
I do not understand what you mean